node_modules/oae-principals/lib/api.user.js

```diff
@@ -493,52 +524,58 @@ var updateUser = module.exports.updateUser = function(ctx, userId, profileFields
             return callback(err);
         }
 
-        // Ensure the provided email, if any, is not used by another user
-        _getUserByEmailIfNecessary(profileFields.email, oldUser.email, function(err, user) {
-            if (err && err.code !== 404) {
+        // If the user wants to change their own email address, we don't change it immediately.
+        // We will make that change once they have verified they own it. We will persist the
+        // desired email address in a separate `emailChange` property and set the `emailVerified`
+        // property to `false`. This will allow the UI to show a nag screen to the user asking
+        // them to confirm their email
+        var isAdmin = ctx.user().isAdmin(oldUser.tenant.alias);
+        var isEmailChange = (!_.isUndefined(profileFields.email) && profileFields.email !== oldUser.email);
+        if (isEmailChange && !isAdmin) {
+            profileFields.emailChange = profileFields.email;
+            profileFields.emailVerified = false;
+            delete profileFields.email;
+        }
```

Comment:
I think the account can still remain active with the previous `email` property until they have verified the change. So deleting the email is not necessary.